cypress/e2e/backoffice/file-manager-attachment/file-list.cy.ts

```diff
@@ -0,0 +1,136 @@
+    const fileManagerAttachmentDetachPage = container.get(FileManagerAttachmentDetachPage);
+
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.clickViewButton();
+
+    fileManagerAttachmentDetachPage.detachFile();
+    fileManagerAttachmentDetachPage.verifySuccessMessage();
+  });
+
+  it('should upload multiple files with size constraints', () => {
+    const fileManagerAttachmentAddPage = container.get(FileManagerAttachmentAddPage);
+
+    // Prepare test files that meet the constraints
+    const testFiles: { fileContent: any; fileName: string; mimeType: string; filePath: string }[] = [];
+
+    cy.fixture('suite/backoffice/file-manager-attachment/test-files/document1.pdf', 'binary')
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'document1.pdf',
+          mimeType: 'application/pdf',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/document1.pdf',
+        });
+      })
+      .then(() => {
+        cy.fixture('suite/backoffice/file-manager-attachment/test-files/image1.jpeg', 'binary');
+      })
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'image1.jpeg',
+          mimeType: 'image/jpeg',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/image1.jpeg',
+        });
+      })
+      .then(() => {
+        cy.fixture('suite/backoffice/file-manager-attachment/test-files/image2.png', 'binary');
+      })
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'image2.png',
+          mimeType: 'image/png',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/image2.png',
+        });
+      })
+      .then(() => {
+        cy.fixture('suite/backoffice/file-manager-attachment/test-files/document2.pdf', 'binary');
+      })
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'document2.pdf',
+          mimeType: 'application/pdf',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/document2.pdf',
+        });
+      })
+      .then(() => {
+        fileManagerAttachmentAddPage.visit();
+        fileManagerAttachmentAddPage.uploadFiles(testFiles);
+        fileManagerAttachmentAddPage.verifyFileUploadConstraints();
+        fileManagerAttachmentAddPage.submitForm();
+        fileManagerAttachmentAddPage.verifySuccessMessage();
+      });
+  });
+
+  it('should successfully delete a file', () => {
+    const fileManagerAttachmentDeletePage = container.get(FileManagerAttachmentDeletePage);
+
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.clickDeleteButton();
+
+    fileManagerAttachmentDeletePage.confirmDelete();
+    fileManagerAttachmentDeletePage.verifySuccessMessage();
```

Comment:
please also add an assertion that table doesn't contain file after deletion, thanks